source/lib-intro.tex

```diff
@@ -1013,41 +1013,46 @@
 {llll}
 \topline
 \tcode{<algorithm>} &
+\tcode{<fstream>} &
```

Comment:
This needs merging / rebasing.

Reply:
Do you want me to rebase (with force-push)? That is my preference; i have no idea what "merge" would do to me.

Reply:
Yes, pushing with --force-with-lease seems fine here. (I think that will probably do bad things to github's comment tracking, though I suppose there's only one way to find out.)